src/metabase/api/geojson.clj

```diff
@@ -165,25 +165,35 @@
     (respond (-> (response/response is)
                  (response/content-type "application/json")))))
 
-(api/defendpoint-async GET "/:key"
+(api.macros/defendpoint :get "/:key"
   "Fetch a custom GeoJSON file as defined in the `custom-geojson` setting. (This just acts as a simple proxy for the
   file specified for `key`)."
-  [{{:keys [key]} :params} respond raise]
-  {key ms/NonBlankString}
-  (when-not (or (custom-geojson-enabled) ((builtin-geojson) (keyword key)))
+  [{k :key, :as _route-params} :- [:map
+                                   [:key ms/NonBlankString]]
+   _query-params
+   _body
+   _request
+   respond
+   raise]
```

Comment:
This is gonna make testing async route functions in isolation more straightforward too. 🚀